### PR TITLE
[release/1.0.0] Mark packages as stable for 1.0.6

### PR DESCRIPTION
**Pull request by weshaggard**
cc @ericstj @bartonjs

### CODE REVIEW

File: pkg/Microsoft.Private.PackageBaseline/stable.packages.props [hunk 1]
```diff
@@ -1397,5 +1397,46 @@
...
+    </StablePackage>
+    <StablePackage Include="runtime.rhel.7-x64.runtime.native.System.Security.Cryptography">
+      <Version>1.0.3</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.ubuntu.14.04-x64.runtime.native.System.Security.Cryptography">
+      <Version>1.0.3</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.ubuntu.16.04-x64.runtime.native.System.Security.Cryptography">
+      <Version>1.0.3</Version>
+    </StablePackage>
```
Comment (bartonjs) [inline pkg/Microsoft.Private.PackageBaseline/stable.packages.props]: Missing X509Certificates?

Comment (weshaggard) [inline pkg/Microsoft.Private.PackageBaseline/stable.packages.props]: yep

Comment (weshaggard) [inline pkg/Microsoft.Private.PackageBaseline/stable.packages.props]: added.

Comment (ericstj) [inline pkg/Microsoft.Private.PackageBaseline/stable.packages.props]: Might be interesting to just point the index update task at something like https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/release/1.0.0/Latest_Packages.txt in order to semi-automate these changes.

Comment (weshaggard) [inline pkg/Microsoft.Private.PackageBaseline/stable.packages.props]: Yes work considering.

**Pull request merged**